lms/templates/help_modal.html

```diff
@@ -254,13 +254,13 @@
             htmlStr = "${_('An error has occurred.')}";
 % if settings.FEEDBACK_SUBMISSION_EMAIL:
             htmlStr += " " + _.template(
-              "${_('Please {link_start}send us e-mail{link_end}.')}",
-              {link_start: '<a href="#" id="feedback_email">', link_end: '</a>'},
-              {interpolate: /\{(.+?)\}/g})
+              "${_('Please {link_start}send us e-mail{link_end}.')}", {interpolate: /\{(.+?)\}/g})(
+                {link_start: '<a href="#" id="feedback_email">', link_end: '</a>'}
+              );
 % else:
             // If no email is configured, we can't do much other than
             // ask the user to try again later
-            htmlStr += "${_('Please try again later.')}";
+            htmlStr += " " + "${_('Please try again later.')}";
```

Comment:
Note to reviewers: Just noticed the missing space. Note that these strings are not i18n friendly, but changing existing code is beyond the scope of this PR.